patches/linux/6.12.22/0001-FIX-net-dsa-mv88e6xxx-Fix-timeout-on-waiting-for-PPU.patch

```diff
@@ -1,15 +1,18 @@
-From 774cf1d3f5b9cfac247c3efcf4eed39d06c675dc Mon Sep 17 00:00:00 2001
+From 412fc6e03cfa025b34791b21a99d2a57a3e55d6b Mon Sep 17 00:00:00 2001
 From: Tobias Waldekranz <tobias@waldekranz.com>
 Date: Tue, 12 Mar 2024 10:27:24 +0100
 Subject: [PATCH 01/28] [FIX] net: dsa: mv88e6xxx: Fix timeout on waiting for
  PPU on 6393X
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
```

Comment:
Do you know why your setup generate these extra changes when `git-format-patch` runs?  I'm thinking we maybe should add something to the script to either always add or never add this ...